build.sh

```diff
@@ -1,6 +0,0 @@
-cd cmd/epoch
-go build -v
-cd ../relayer
-go build -v
-cd ../cache
-go build -v
```

Comment:
Could we keep the build.sh, it is easy for developers to build and use.

Reply:
I think the `build.sh`'s code is not good...and `makefile` has the same function, for example:

Just run `make` under `/ethashproof` will build binaries to `target/`
+ target/relayer
+ target/cache
+ target/epoch